network/sendMail.ts

```diff
@@ -0,0 +1,51 @@
+import {Platform} from 'react-native';
+
+import {Logger} from 'browser-bunyan';
+
+import * as FileSystem from 'expo-file-system';
+import * as Linking from 'expo-linking';
+import * as MailComposer from 'expo-mail-composer';
+
+interface SendMailProps {
+  to: string;
+  subject: string;
+  body?: string;
+  attachments?: string[];
+  logger?: Logger;
```

Comment:
Why should we allow this to be undefined?

Reply:
no real reason I guess. might make testing easier but since I didn't write any tests... 🙈